### PR TITLE
Add initial "watch" support.

### PR DESCRIPTION
**Pull request by davidlehn**
Requires bedrock-views patch.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -95,19 +116,15 @@ bedrock.events.on('bedrock-views.optimize.run', (options, callback) => {
  *          [output] filename to output
  *          [paths] paths for output (local and public)
  *          [configs] an array of webpack configs to merge
+ * FIXME
```
Comment (mattcollier) [inline lib/index.js]: what is this about?

File: lib/index.js [hunk 2]
```diff
@@ -299,19 +326,20 @@ api.optimize = function(options, callback) {
...
       }
-      return callback(err);
+      return err;
     }
 
     const info = stats.toJson();
 
     if(stats.hasErrors()) {
-      logger.error('webpack errors:\n' + info.errors);
+      const s = info.warnings.toString();
```
Comment (mattcollier) [inline lib/index.js]: should this be `info.errors`?  I see `info.warning` being used just below.

File: lib/index.js [hunk 3]
```diff
@@ -320,43 +348,72 @@ api.optimize = function(options, callback) {
...
-    let statsString = stats.toString({
-      chunks: false,
-      colors: true
-    });
-    logger.info(`webpack stats:\n${statsString}\n`);
-    */
+    // FIXME: add new option to control this output
+    // reuse stats option
+    if(command.webpackStats === 'true') {
+      const s = stats.toString({
```
Comment (mattcollier) [inline lib/index.js]: one line?

File: lib/index.js (hunk 1 above)
Comment (davidlehn) [reply lib/index.js]: What's the proper text to document a callbackified async function?

File: lib/index.js (hunk 2 above)
Comment (davidlehn) [reply lib/index.js]: Copy & paste error.  This flow has never been well tested.  Might be wrong in watch mode.

File: lib/index.js (hunk 3 above)
Comment (davidlehn) [reply lib/index.js]: You mean pull the options object into one line?

Comment (mattcollier) [reply lib/index.js]: yes, that's what I'm suggesting.

**Comment by dlongley** [comment]
@davidlehn -- this seems like it was superseded, can we close?

**Comment by davidlehn** [comment]
All these commits are in master now. Closing.  Leaving the large number of watch* branches around for now since there may be some code out there still depending on them.

**Pull request closed without merging**